Lil-Peep/src/scripts/theme.js

```diff
@@ -52,3 +52,49 @@ $(document).ready(function() {
     document.documentElement.className = document.documentElement.className.replace('supports-no-cookies', 'supports-cookies');
   }
 });
+/*
```

Comment:
remove this comments

Reply:
En mi opinion no deberia incluir comentarios de codigo que no usas.